scaling_performance/optimizing_storage.adoc

```diff
@@ -11,25 +11,185 @@
+| Recommended
+| Recommended
+| Not configurable
+| Not configurable
+| Not configurable footnoteref:[disclaimer5, Object storage is not consumed through {product-title}'s PVs/persistent volume claims (PVCs). Apps must integrate with the object storage REST API.]
+|===
+
+[NOTE]
+====
+A scaled registry is an {product-title} registry where 3 or more pod replicas are running.
+====
+
+[[application-storage-recommendations]]
+==== Specific Application and Storage Recommendations
+
+[[registry]]
+===== Registry
+
+In a non-scaled/high-availability (HA) {product-title} registry cluster deployment:
+
+* The preferred storage technology is object storage followed by block storage. The
+storage technology does not need to support ReadWriteMany (RWX) access mode.
+* The storage technology must ensure read-after-write consistency. All NAS storage (excluding CNS/CRS GlusterFS as it uses an object storage interface) are not
+recommended for {product-title} Registry cluster deployment with production workloads.
+* While hostPath volumes are configurable for a non-scaled/HA {product-title} Registry cluster deployment, they are not recommended.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back {product-title} registry at production workloads.
+====
+
+[[scaled-registry]]
+===== Scaled Registry
+
+In a scaled/HA {product-title} registry cluster deployment:
+
+* The preferred storage technology is object storage. The storage technology must support RWX access mode and must ensure read-after-write consistency.
+* File storage and block storage are expressly not recommended for a scaled/HA {product-title} registry
+cluster deployment with production workloads.
+* All NAS storage (excluding CNS/CRS GlusterFS as it uses an object storage interface) are
+not recommended for {product-title} Registry cluster deployment with production workloads.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back {production-title} scaled/HA registry at production workloads.
+====
+
+[[metrics]]
+===== Metrics
+
+In an {product-title} hosted metrics cluster deployment:
+
+* The preferred storage technology is block storage.
+* It is not recommended to use all NAS storage (excluding CNS/CRS GlusterFS as it uses a block storage interface from iSCSI) for hosted metrics cluster deployment with production workloads.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back hosted metrics at production workloads.
+====
+
+[[logging]]
+===== Logging
+
+In an {product-title} hosted logging cluster deployment:
+
+* The preferred storage technology is block storage.
+* It is not recommended to use all NAS storage (excluding CNS/CRS GlusterFS as it uses a block storage interface from iSCSI) for hosted metrics cluster deployment with production workloads.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back hosted logging at production workloads.
+====
+
+[[applications]]
+===== Applications
+
+Application use cases vary from application to application:
```

Comment:
This needs more - something like "the following are some examples" or some such lead into the list.